app/views/observation_matrices/export/nexus/_nexus.html.erb

```diff
@@ -20,12 +20,13 @@ Begin characters;
 Title qualitative_characters;
 Dimensions Nchar=<%= descriptors.size -%>;
 Format
-<%= as_file ? '  ' : nb -%>datatype = STANDARD  
-<%= as_file ? '  ' : nb -%>missing = ?  
-<%= as_file ? '  ' : nb -%>symbols = "0 1 2 3 4 5 6 7 8 9 A B C D E F G H I J K L M N P Q R S U V W X Y Z !"; 
+<%= as_file ? '  ' : nb -%>datatype = STANDARD
+<%= as_file ? '  ' : nb -%>gap = -
+<%= as_file ? '  ' : nb -%>missing = ?
+<%= as_file ? '  ' : nb -%>symbols = "0 1 2 3 4 5 6 7 8 9 A B C D E F G H I J K L M N P Q R S U V W X Y Z !";
```

Comment:
Side note: we could improve the `symbols` line by referencing the observed states in the data, this would be a fairly big improvement.